electron/locale/strings-it.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+let string = {};
+
+string.menuConversation = 'Conversazione';
+string.menuStart = 'Avvia';
+string.menuEdit = 'Modifica';
+string.menuWindow = 'Finestra';
+string.menuView = 'View';
+string.menuHelp = 'Aiuto';
+string.menuStartup = 'Apri Wire all\'avvio del sistema';
```

Comment:
replace it with the above instead of escaping the fix from https://github.com/wireapp/wire-webapp/pull/482 is coming here as well